tests/DataObjectTest.py

```diff
@@ -91,10 +91,6 @@ class A(DataObject):
             a = DatatypeProperty()
             properties_are_init_args = True
 
-        class B(A):
```

Comment:
This test can be removed. I think when it was written, I had some question about how attribute access would work with the parent class, but at this point it's pretty vanilla.